taker-frontend/src/components/Tour.tsx

```diff
@@ -0,0 +1,194 @@
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { Link, Text } from "@chakra-ui/react";
+import { Steps } from "intro.js-react";
+import * as React from "react";
+import { useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { FAQ_URL } from "../App";
+import cfd101 from "../images/CFD_101_light_bg.svg";
+import confirmationDialog from "../images/confirmation_dialog.png";
+import itchyGuyWelcome from "../images/ItchyGuyWelcome.svg";
+
+export const Tour = () => {
+    const tourSteps = [
+        {
+            title: "G'Day Hodlers!",
```

Comment:
🙃
A hodler does not trade, he hodls 😬
```suggestion
            title: "G'Day Bitcoiner!",
```

Reply:
Fair enough! I guess strictly hodl really just stands for "keep your money under the pillow".
I saw it more from the perspective of Bitcoin in Bitcoin out, I want to only grow my Bitcoin.

Reply:
this is a good question. under my naive assumption, trading *in* bitcoin, is still HODLing bitcoin, as you don't get rid of it! but I defer to your judgements, as I'm a relative newbie in this space!